README.md

```diff
@@ -55,7 +55,14 @@ support other orchestration environments in the future.
 
 NOTE:
 
-- **`csiv0.3`** is deprecated with release of **`csi v1.1.0`**
+The supported window of Ceph CSI versions  is known as "N-1":
+(N (Latest major release) - 1 (major versions)).
+
+For example, if Ceph CSI latest major version is `3.0.0` today, support is
+provided for the versions above `2.0.0`. If customers are running an unsupported
+Ceph CSI version, they will be asked to upgrade when requesting support for the
+cluster. Clusters running unsupported Ceph CSI releases are not covered by the
+support policies.
```

Comment:
What `support policies`? Sounds like there is an SLA of some kind somewhere... If a policy is documented, link to it please.

Reply:
it was a general comment , however removed that line to avoid confusion.